src/main/java/org/terracotta/build/plugins/AngelaPluginExtension.java

```diff
@@ -0,0 +1,7 @@
+package org.terracotta.build.plugins;
+
+import org.gradle.api.provider.ListProperty;
+
+public interface AngelaPluginExtension {
+  ListProperty<String> getSpawnedProcessConfig();
```

Comment:
If we want to add an extension to control the options of the Angela plugin we should probably be using `JavaForkOptions` which is designed for this.